mod/photos.php

```diff
@@ -1141,9 +1141,7 @@ function photos_content(App $a)
 						'$qcomment' => $qcomment,
 						'$rand_num' => Crypto::randomDigits(12),
 						// Dropzone
-						//'$max_imagesize'       => DI::config()->get('system', 'maximagesize'),
-						// don't know, if DI::config.. does not work here, so it is set to a manual value
-						'$max_imagesize' => 100000,
+						'$max_imagesize'       => DI::config()->get('system', 'maximagesize'),
```

Comment:
So, going back to my initial request, you need the call to `Strings::getBytesFromShortHand` because the `system.maximagesize` value accepts both integer numbers of bytes and string shorthands like `2M` or `2G`. Unless you know that the Dropzone library accepts both, you have to convert the config value to always be an integer, and this is what `Strings::getBytesFromShortHand` does.

Reply:
I understand.

Will change this.

Reply:
> So, going back to my initial request, you need the call to `Strings::getBytesFromShortHand` because the `system.maximagesize` value accepts both integer numbers of bytes and string shorthands like `2M` or `2G`. Unless you know that the Dropzone library accepts both, you have to convert the config value to always be an integer, and this is what `Strings::getBytesFromShortHand` does.

Again... this is not working...

I have both tried:
```
~                         //'$max_imagesize'       => Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize'));
+                         '$max_imagesize'       => \Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize'));
```

with and without `use Friendica\Util\Strings;` in the header.

It produces this:

`2023-03-13T19:58:42Z index [CRITICAL]: Uncaught Exception ParseError: "syntax error, unexpected token ";", expecting "]"" at /var/www/soc.schuerz.at/src/Module/Post/Edit.php line 189 {"exception":"ParseError: syntax error, unexpected token \";\", expecting \"]\" in /var/www/soc.schuerz.at/src/Module/Post/Edit.php:189\nStack trace:\n#0 /var/www/soc.schuerz.at/vendor/composer/ClassLoader.php(322): Composer\\Autoload\\includeFile()\n#1 [internal function]: Composer\\Autoload\\ClassLoader->loadClass()\n#2 /var/www/soc.schuerz.at/vendor/level-2/dice/Dice.php(73): is_subclass_of()\n#3 /var/www/soc.schuerz.at/vendor/level-2/dice/Dice.php(93): Dice\\Dice->getRule()\n#4 /var/www/soc.schuerz.at/src/App/Router.php(353): Dice\\Dice->create()\n#5 /var/www/soc.schuerz.at/src/App.php(691): Friendica\\App\\Router->getModule()\n#6 /var/www/soc.schuerz.at/index.php(52): Friendica\\App->runFrontend()\n#7 {main}"} - {"file":null,"line":null,"function":null,"request-id":"640f8072c469e","uid":"51f411","process_id":1267833}
`

Reply:
I tried the following:
```
Strings::getBytesFromShorthand('5G');
Strings::getBytesFromShorthand(5000000);
```
Both gave me the same error, that the line expects "]" instead of ";"

Reply:
In src/Model/Photo.php is the same call... and obvious it works there.

Someshing ist missing?

Reply:
Yes, template variables are declared inside of an array which values should be separated by commas, not semi-colon. The snippet I provided to you only works if it's on its own line, not when it is used inside an array declaration.
```suggestion
						'$max_imagesize'       => \Friendica\Util\Strings::getBytesFromShorthand(DI::config()->get('system', 'maximagesize')),
```